packages/sdks/output/svelte/src/blocks/columns/columns.svelte

```diff
@@ -0,0 +1,103 @@
+<script>
+    
```

Comment:
I assume all the wacky spacing in here is due to svelte not playing well with prettier?

Reply:
Exactly, the svelte prettier plugin was crashing constantly so I disabled it for now. 

It's on my list of lower-priority things to investigate once the SDK is in a working state.